srcjs/handlers/updateApp.js

```diff
@@ -5,6 +5,42 @@ $(function() {
 
   // Update app instance params (see updateF7App)
   Shiny.addCustomMessageHandler("update-app", function(message) {
+    // If color, dark or theme is changed, we need to update it
+    if (message.hasOwnProperty("color")) {
+      app.setColorTheme(message.color);
+    }
+
+    if (message.hasOwnProperty("dark")) {
+      if (message.dark) {
+        $("html").addClass("dark");
+        $("html").removeClass("light");
+        $("body").addClass("dark");
+        $("body").removeClass("light");
+      } else {
+        $("html").addClass("light");
+        $("html").removeClass("dark");
+        $("body").addClass("light");
+        $("body").removeClass("dark");
+      }
+    }
+
+    if (message.hasOwnProperty("theme")) {
+      if (message.theme === "md") {
+        $("html").addClass("md");
+        $("html").removeClass("ios");
+        $(".tab-link-highlight").show();
+      } else if (message.theme === "ios") {
+        $("html").addClass("ios");
+        $("html").removeClass("md");
+        $(".tab-link-highlight").hide();
+      }
+    }
+
+    // Remove color, dark and theme params from message
+    delete message.color;
+    delete message.dark;
+    delete message.theme;
```

Comment:
Why do we need to remove the theme from the config? It is a valid parameters: https://framework7.io/docs/app#param-theme

Reply:
We don’t have to remove it necessarily, but those 3 params are covered above and there’s no need to send them again. It wouldn’t hurt though to leave them

Reply:
Removed in latest commit